components/emoji_picker/modules/emoji_selector.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="d-emoji-picker__selector">
+    <!--    Emoji Selector -->
+    <div
+      v-for="emoji in EMOJI_LIST"
+      :key="emoji"
+      class="d-emoji-picker__selector--emoji-list"
+    >
+      <p
+        class="d-emoji-picker__selector--emoji-list__tab"
+      >
+        {{ emojiFilter ? 'Search results' : emoji.tab }}
+      </p>
+
+      <div>
+        <dt-emoji
+          v-for="emoji in filteredEmojiList"
+          :key="emoji.code"
+          class="d-emoji-picker__selector--emoji-list__emoji"
+          :code="emoji.code"
+          @mouseover="$emit('emoji-data', emoji)"
+          @mouseleave="$emit('emoji-data', null)"
+          @click="$emit('selected-emoji', emoji)"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { DtEmoji } from '@/components/emoji';
+import { emojiJson } from '@/common/emoji';
+
+// TO DO EMOJI JSON = EMOJI LIST AND UTILITY FUNCTIONS
+
+import { computed } from 'vue';
+const props = defineProps({
+  /**
+   * The filter to apply to the emoji list
+   * @type {String}
+   * @default ''
+   */
+  emojiFilter: {
+    type: String,
+    default: '',
+  },
+
+  /**
+   * The skin tone to apply to the emoji list
+   * @type {String}
+   * @default null
+   */
+  skinTone: {
+    type: Number,
+    default: null,
+  },
```

Comment:
How do you plan to filter skin tone using the existing json data? You don't have to write code or anything but a couple sentences about the method you intend on using is useful to figure out at this point.

Reply:
Done my research above I still need to figure out this kind of methods, because the complexity and also the perform impact.